examples/resources/onelogin-connector.yaml

```
@@ -3,14 +3,24 @@ version: v2
 metadata:
   name: OneLogin
 spec:
-  acs: https://<cluster-url>/v1/webapi/saml/acs
-  # Controls whether IdP-initiated SSO is allowed. If false, all such requests will be rejected with an error.
-  allow_idp_initiated: false
+  acs: https://<cluster-url>/v1/webapi/saml/acs/onelogin
   attributes_to_roles:
-    - {name: "groups", value: "admin", roles: ["editor"]}
-    - {name: "groups", value: "dev", roles: ["access"]}
+  - name: groups
+    roles:
+    - editor
+    value: admin
+  - name: groups
+    roles:
+    - access
+    value: dev
+  audience: https://tp.alexf.xyz:443/v1/webapi/saml/acs/onelogin
```

```suggestion
  audience: https://<cluster-url>/v1/webapi/saml/acs/onelogin
```

We should be consistent in this file - there's a generic placeholder "variable" in one place, a specific alexf.xyz domain in another place, and a teleport.example.com in another place.

Also, `<cluster-url>` is probably not the right name anyway, as it's not a URL that goes there. Maybe `<proxy-hostname>` instead?

---

This was a flub, that's my domain I was testing with, will scrub from the history.